steering_controllers/ackermann_steering_controller/src/ackermann_steering_controller.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2022, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ackermann_steering_controller/ackermann_steering_controller.hpp"
+
+namespace ackermann_steering_controller
+{
+//using AckermannSteeringController = steering_controllers::SteeringControllers;
+AckermannSteeringController::AckermannSteeringController()
+: steering_controllers::SteeringControllers()
+{
+}
+
+controller_interface::CallbackReturn AckermannSteeringController::configure_odometry()
+{
+  params_ = param_listener_->get_params();
```

Comment:
It could be that here makes sense to have two parameter files:

`steering_controllers_parameters.yaml`
`ackermann_steering_parameters.yaml`

Then the `SteeringParameters` are loaded in the parent class and here only ackermann.

so we would have here something like this:

```
params<AckermannSteeringParams> ackermann_params = ackermann_param_listener_->get_param();

// get all joints and set them in odometry
// calculate params as now form those two values
// set them to odometry
```